src/Microsoft.DotNet.Tools.Compiler/CompilerUtil.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
I am using this class to hold the functionality that needs to be available to both build and compile. Currently I'm using it to learn and collect what the common interface is. Ideally, this communication should be done via something more first class than a utility class (an interface maybe?).

For now, I'd rather keep it like this until we get a better idea on how the 2 verbs need to communicate.